src/libsync/networkjobs.h

```diff
@@ -420,6 +420,12 @@ public slots:
      * @param statusCode - the OCS status code: 100 (!) for success
      */
     void etagResponseHeaderReceived(const QByteArray &value, int statusCode);
+    
+    /**
+     * @brief desktopNotificationStatusReceived - signal to report if notifications are allowed
+     * @param status - set desktop notifications allowed status 
+     */
+    void allowDesktopNotificationsChanged(const bool isAllowed);
```

Comment:
This strikes me as strange design wise that this sits on the very general JsonApiJob... I'd have expected the corresponding header to be checked in UserStatus (IIRC you should have access to the reply there).